modules/tinymce/src/core/test/ts/browser/util/ToolsTest.ts

```diff
@@ -12,7 +12,9 @@ describe('browser.tinymce.core.util.ToolsTest', () => {
 
   context('create', () => {
     it('TINY-7358: Multiple calls to create should create different objects', () => {
+      // eslint-disable-next-line @tinymce/prefer-fun
```

Comment:
Why?

Reply:
This is a class constructor, it can't use constants.